system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -20,6 +20,10 @@
         @include context-menu-open-left;
```

Comment:
Looks like we'll need a similar rule/include for widest:
![screen shot 2018-02-06 at 7 51 24 pm](https://user-images.githubusercontent.com/438537/35897686-4a92b4c0-0b77-11e8-8261-bbac7ba24386.png)

Reply:
Also for top sites too.